src/Responder/FormattedResponder.php

```diff
@@ -14,11 +14,16 @@
 use Equip\Resolver\ResolverTrait;
 use Relay\ResolverInterface;
 
-class FormattedResponder extends Dictionary implements ResponderInterface
+class FormattedResponder extends SortedDictionary implements ResponderInterface
 {
     use ResolverTrait;
 
     /**
+     * @var callable
+     */
+    protected $sorter = 'arsort';
```

Comment:
Exactly. That's what we want, hence the PR. All of the formatters should be sorted from highest to lowest quality. Before it was order added, which could be incorrect.